### ISSUE TITLE
💡 [REQUEST] - Clarification on dosage dose UCUM as UOM - ANFISK-334

### ISSUE THREAD

**Issue by julsas** [gematik/ISiK-Medikation#133]
### Summary

Need to clarify if other code systems besides UCUM should be allowed for `MedicationAdministration.dosage.dose` and `Dosage.doseAndRate.doseQuantity` for MedicationRequest and MedicationStatement.

ISiK-3 has Quantity.system = http://unitsofmeasure.org. Should other code systems be allowed for concepts like Tablet from EDQM?
ISiK-4 introduces the Quantity data type profile Medication Quantity which mentions SNOMED as an alternative to UCUM. But Quantity.system is still set to UCUM.
This could this be a use case for http://hl7.org/fhir/R4/extension-iso21090-pq-translation.html 

### Common Examples

Use case of a Medication with a form:
```json
    "form": {
        "coding":  [
            {
                "code": "10219000",
                "system": "http://standardterms.edqm.eu",
                "display": "Tablet"
            }
        ]
    },
```
should it be possible to have a MedicationAdministration of that Medication with:
```json
  "dosage": {
    "dose": {
      "system": "http://standardterms.edqm.eu",
      "value": 1,
      "code": "10219000",
      "unit": "Tablet"
    }
  }
```
?

### Drawbacks

Adds extra complexity when different code systems are allowed.
US Core is UCUM only: https://build.fhir.org/ig/HL7/US-Core/StructureDefinition-us-core-medicationrequest.html

**Comment by f-peverali** [comment]
hallo @julsas. Danke für die Anfrage. Diskutieren wir intern und dann sicher auch in einer AG.

**Comment by f-peverali** [comment]
@patrick-werner und @lhitc wie ordnet ihr das ein?

**Comment by patrick-werner** [comment]
Der Kommentar auf MedicationQuantity ist ein Überbleibsel einer Überlegung die wir dann aber nicht umgesetzt hatten.
Der letzte Stand der Diskussion war hier ggfs. zukünftig eine Extension zu nutzen, da Quantity mit mehreren unterschiedlichen Kodierungen sehr komplex in der Implementierung sind und die internationale Kompatibilität hier wichtig ist.

Damit würde man die Tablette in UCUM weiterhin mit '1' kodieren, aber in der Extension mittels SCT oder EDQM diese '1' weiter beschreiben. Diese Extension würde dann aber nur ein Coding enthalten, keine Quantity wie in der oben erwähnten Extension.

@julsas Wäre das für dich eine akzeptable Lösung?
Die KBV hatte hier stets SCT Values gefordert, dein Beispiel nutzt EDQM. Eine einheitliche Kodierung wäre hier wünschenswert. SCT wäre global "kompatibler" und auch meine Präferenz.

**Comment by julsas** [comment]
Ja, das ist eine akzeptable Lösung. Auch SCT ist gut. Bei EDQM ist irgendwie auch unklar, ob das relevant wird für den EHR-Bereich, oder nur bei Pharma und Arzneimittelzulassung angewendet wird.

**Comment by patrick-werner** [comment]
Da der fix bereits umgesetzt wurde schließe ich hier. Zusätzliche SCT Kodierungen werden wir in einem AK Meeting besprechen
